client/javascripts/sagas/auth.js

```diff
@@ -0,0 +1,19 @@
+import { call, takeEvery } from 'redux-saga/effects';
+import * as api from 'api';
+import {
+  REQUEST_SIGN_OUT,
+} from 'actions/authActionCreators';
+import { browserHistory } from 'react-router';
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'react-router'. (import/no-unresolved)
* Error - Missing file extension for "react-router" (import/extensions)

<!-- Comment by [SideCI](https://sideci.com) -->